### PR TITLE
Implement Minimum Password Length Check on Update Password Page #590

### PR DESCRIPTION
**Pull request by ShubhamSingh-S34**
min current password required to send PUT request to '/api/password' is now changed to 7 which is the minimun length of password required while signing up. client side validation is done . Tell if server side validation is required.

password length 6. Button is disabled
Screenshot (52)

password length 7 . Button is enabeled
Screenshot (53)

### CODE REVIEW

**Comment by NathanTarbert** [comment]
Hi @ShubhamSingh-S34, would you mind modifying your submission?  [NIST](https://pages.nist.gov/800-63-3/sp800-63-3.html) recommends a minimum password length of 8. 
I am adding this to #590.

**Comment by ShubhamSingh-S34** [comment]
Done

File: components/account/UpdatePassword.tsx [hunk 1]
```diff
@@ -8,8 +8,8 @@ import { Card, InputWithLabel } from '@/components/shared';
 import { defaultHeaders } from '@/lib/common';
 
 const schema = Yup.object().shape({
-  currentPassword: Yup.string().required(),
-  newPassword: Yup.string().required().min(7),
+  currentPassword: Yup.string().required().min(8),
```
Comment (devkiran) [inline components/account/UpdatePassword.tsx:11]:
```suggestion
  currentPassword: Yup.string().required(),
```
Remove the length check on the `currentPassword`.

**Review by devkiran — CHANGES_REQUESTED** [review]

**Comment by devkiran** [comment]
Already covered in another PR.

**Pull request closed without merging**